src/gtk/window.cpp

```diff
@@ -225,6 +230,11 @@ static wxWindowGTK *gs_deferredFocusOut = nullptr;
 GdkEvent    *g_lastMouseEvent = nullptr;
 int          g_lastButtonNumber = 0;
 
+// globals variables needed for x11 raw key codes to latin chars transformations
+xkb_context *ctx = nullptr;
+xkb_keymap *keymap = nullptr;
+xkb_state *state = nullptr;
```

Comment:
We really should use `g_` prefix for them. Or, perhaps, put them all in some `g_xkbData` struct and then use the names without prefix for its fields.